docs/recipes/auth/README.md

```diff
@@ -0,0 +1,95 @@
+# Running Lighthouse on Authenticated Pages with Puppeteer
+
+If you just want to view the code, see [example-lh-auth.js](./example-lh-auth.js).
+
+## The Example Site
+
+There are two pages on the site:
+
+1. `/` - the homepage
+2. `/dashboard`
+
+The homepage shows the login form, but only to users that are not signed in.
+
+The dashboard shows a secret to users that are logged in, but shows an error to users that are not.
+
+The server responds with different HTML for each of these pages and session states, so there are four different pages that must have passable Lighthouse SEO scores.
```

Comment:
> so there are four different pages that must have passable Lighthouse SEO scores.

this seems a little uncharacteristic for us. we don't really talk about what pages should be passing and where. (there's also nuance in this topic of auth + seo, so if you're cool with nuking this phrase we don't even need to get into this :)

Reply:
OK with just deleting the phrase.

but made me think, maybe I should assert on perf instead?

Reply:
nvm the integration test isn't here any more so no need to talk about this